packages/shared/src/components/RecommendedMention.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { UserShortInfo } from './profile/UserShortInfo';
+
+interface RecommendedUser {
+  image: string;
+  name: string;
+  username: string;
+}
+
+interface RecommendedMentionProps {
+  users: RecommendedUser[];
+  selected: number;
+  onClick?: (username: string) => unknown;
+}
+
+export function RecommendedMention({
+  users,
+  selected,
+  onClick,
+}: RecommendedMentionProps): ReactElement {
+  if (!users?.length) {
+    return null;
+  }
+
+  return (
+    <ul
+      className="flex overflow-hidden flex-col rounded-16 border border-theme-divider-secondary text-theme-label-primary"
+      style={{ width: '15rem' }}
```

Comment:
Perhaps we can change this to be a JIT class?

Reply:
Ah yes probably, what mostly works is stopping the server, removing the .next file in your webapp and re-running.

Reply:
Worked! After re-run and deleting in-between. Thanks!

Reply:
@rebelchris looks like on our build it doesn't generate the arbitrary value. Have you had any situation where you were able to make it work in production?

Preview deployment:
![Screen Shot 2022-03-08 at 8 02 16 PM](https://user-images.githubusercontent.com/13744167/157234523-84cc7a0c-ca25-43e0-877c-3a23bbc343ee.png)

Reply:
I haven't seen it be an issue in production.
But don't let this block you, rather move back to style solution and add a separate ticket for this 😮‍💨

Reply:
Hmm. Let me clear my cache once I come back, could be just a cache issue on my end if it will be working on yours.